pkgs/tools/misc/pridefetch/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, lib, python39, zip }: let
+  version = "1.0.0";
+  sha256 = "sha256-/o4er8bO/3HUFXzP+sC+5DYv9EwmxW05o1RT5fEulEg=";
+
+  name = "pridefetch";
+  src = fetchFromGitHub {
+    owner = "SpyHoodle";
+    repo = name;
+    rev = "v" + version;
+    inherit sha256;
+  };
+in stdenv.mkDerivation {
+  inherit name version;
+  buildInputs = [
+    (python39.withPackages (pythonPackages: with pythonPackages; [
+      distro
+    ]))
+    zip
+  ];
+  dontUnpack = true;
+  buildPhase = ''
+    mkdir $out
```

Comment:
Typically build products aren't moved to `$out` in the build phase.

Here's my proposed division of the work between the two phases (I'd also be fine with putting everything in `installPhase`, given that I can't decide whether "zipping a directory and slapping a shebang on top" qualifies as "build" or "install"):

- in the build phase, create the `pridefetch` executable in the current directory (and make it executable).

- in the install phase, `mkdir -p $out/bin` and move `pridefetch` there.

And whatever you do, don't forget to add [phase hooks](https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/missing-phase-hooks.md).